src/toil/jobStores/utils.py

```diff
@@ -109,3 +109,61 @@ def single_attempt():
             yield
 
         yield single_attempt()
+
+
+def getJobStoreCls(locator):
+    """
+    Returns the concrete job store class of the given job store locator string. Note that the
+    locator must be prefixed with that name of a supported job store. If the locator string is not
+    properly prefixed or is prefixed with an unsupported job store name a run time error is raised.
+
+    The syntax of a job store locator string prefixed with a job store name is as follows:
+        <job store name>:<job store locator>
+        where <job store name> is one of the following supported job store names:
+            - 'file'
+            - 'aws'
+            - 'azure'
+            - 'google'
+
+    For details about job store locator syntax see the _processLocator method of the relevant
+    concrete job store class.
+
+    Note that for backwards compatibility a job store locator that is prefixed with a '/' or '.'
+    without a colon is considered a valid file job store locator.
```

Comment:
Explain what the equivalent, fully qualified syntax is, i.e. `/foo` is equivalent to `file:/foo` and `./foo` is equivalent to `file:./foo`. Remove the `without the colon` because that's not actually true.